src/components/catalog/catalog.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import List from './list';
+
+const Container = styled.div`
```

Comment:
Styled component shouldn't be in a separate file with all styled components ?

Reply:
it's a choice of taste, I like this minimalistic approach

Reply:
I don't want to always keep opened 100500 files in my editor )